apps/web-giddh/src/app/invoice/preview/invoice.preview.component.html

```diff
@@ -67,54 +103,23 @@
                             </ul>
                         </div>
                     </div>
-                    <button class="btn btn-default desktop-view-btn mr-r15" [tooltip]="commonLocaleData?.app_export"
+                    <button class="btn btn-default desktop-view-btn" [tooltip]="commonLocaleData?.app_export"
                         (click)="exportCsvDownload()" [ladda]="exportInvoiceRequestInProcess$ | async"
                         *ngIf="selectedVoucher === 'sales' && selectedInvoicesList.length && isExported"
                         data-spinner-size="20px">
                         <span class="icon-export-icon"></span>
                     </button>
                     <!-- E-way bill generation and export to excel for a single selected item in invoice ends sizes -->
                     <!-- Section to create New Sales Invoice, Cash Invoice, Credit Note and Debit Note -->
+                    <div class="advance-icon cursor-pointer align-items-center btn btn-primary"
+                    [tooltip]="commonLocaleData?.app_advance_search" (click)="toggleAdvanceSearchPopup(false)">
+                    <span class="pd-r05">Filter</span> <span class="icon-advance-filter"></span>
```

Comment:
Do add multi-lingual labels and avoid hard coded values

Reply:
done